text/latex.go

```diff
@@ -49,23 +56,21 @@ func (hdlr Latex) Box(txt string, fnt vg.Font) (width, height, depth vg.Length)
 	height = vg.Length(box.Height())
 	depth = vg.Length(box.Depth())
 
-	return width, height, -depth
+	dpi := vg.Length(hdlr.dpi() / latexDPI)
+	return width * dpi, height * dpi, depth * dpi
 }
 
 // Draw renders the given text with the provided style and position
 // on the canvas.
 func (hdlr Latex) Draw(c *draw.Canvas, txt string, sty draw.TextStyle, pt vg.Point) {
-	dpi := hdlr.DPI
-	if dpi == 0 {
-		dpi = 72
-	}
+	dpi := hdlr.dpi() / latexDPI
```

Comment:
This isn't needed until L85. Can it be moved down to there?

Reply:
done.